### PR TITLE
feat: goerli default network

### PR DESCRIPTION
**Pull request by superical**
## Summary

As ropsten and rinkeby eventually get deprecated, this PR makes goerli the default testnet. This PR also sorts the ropsten and rinkeby networks down on the list.

image

## Changes

- Make goerli the default testnet
- Sort network order

## Issues

- https://www.pivotaltracker.com/story/show/183311335

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "blockchain:contracts": "node --experimental-modules ./tests/e2e/setup-contracts.mjs",
     "build": "run-s check-types build:css build:app",
     "build:dev": "cross-env NODE_ENV=development npm run build",
-    "build:test": "cross-env NODE_ENV=test npm run build",
+    "build:test": "cross-env NODE_ENV=test NET=ropsten npm run build",
```
Comment (simboonlong) [inline package.json:14]: we will need to re-issue our fixtures (if any) in goerli later?

Comment (superical) [reply package.json:14]: Yea, ropsten will be sunset soon by year end.

Comment (simboonlong) [reply package.json:14]: mm ok lemme go through the testcafe fixtures and reissue them so we can avoid `ropsten` here. might need some time

**Review by simboonlong — APPROVED** [review]
figured out maybe i shouldn't block this pr lets go first. test fixtures stuff i'll raise a separate pr later

**Pull request merged**